docker/docker-compose.yml

```diff
@@ -28,11 +28,10 @@ services:
       - PUID=1000
       - PGID=1000
     volumes:
-      - ferdium-database-vol:/data
+      - /C/Users/SpecialAro/Desktop/random/data:/data
```

Comment:
I guess this shouldn't stay that way :stuck_out_tongue:

Reply:
ahahah yes, I've changed it xD Thank you!